sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -2706,33 +2706,6 @@ class HiveDDLSuite
     }
   }
 
-  test("SPARK-30098: create table without provider should " +
```

Comment:
It didn't exist before SPARK-30098, and this test depends on SPARK-30098.

Reply:
These two tests are depending on the config, hence I'd rather not restore these tests.